dist/cookie/{{cookiecutter.project_slug}}/public/index.html

```diff
@@ -3,9 +3,26 @@
 
     <head>
         <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"
```

Comment:
> maximum-scale=1, user-scalable=0

Curious why we need those? The HTML spec says this about those properties:

**maximum-scale**

    Controls how much zoom in is allowed on the page. Any value less than 3 fails accessibility. Minimum: 0.1. Maximum: 10. Default: 10. Negative values: ignored.

**user-scalable**

    Controls whether zoom in and zoom out actions are allowed on the page. Valid values: 0, 1, yes, or no. Default: 1, which is the same as yes. Setting the value to 0, which is the same as no, is against Web Content Accessibility Guidelines (WCAG).
    
 Seems like we might not want to use a value below 3 for `maximum-scale`, and we should allow `user-scalable`. Perhaps we can remove those altogether? I don't see any difference with our without those on the page.

Reply:
ok no problem, i could go back to the prev version. i usually set the index.html with this content to prevent zoom issues. going to the prev version 🚀